evap/static/ts/src/csrf-utils.ts

```diff
@@ -15,7 +15,11 @@ const csrftoken = getCookie("csrftoken")!;
 
 function isMethodCsrfSafe(method: string): boolean {
     // these HTTP methods do not require CSRF protection
-    return ["GET", "HEAD", "OPTIONS", "TRACE"].includes(method);
+    return ["GET", "HEAD", "OPTIONS", "TRACE"].includes(method.toUpperCase());
```

Comment:
@He3lixxx said it earlier today, and I agree: We should allow-list the methods that do get the token and not the other way around; if we ever fail to send a token, django will, I hope, happily tell us about it :^)